engo_js.go

```diff
@@ -293,22 +295,22 @@ func loadFont(r Resource) (*truetype.Font, error) {
 	return freetype.ParseFont(ttfBytes)
 }
 
-type ImageObject struct {
+type HtmlImageObject struct {
```

Comment:
Basically with the `texImage2D` method, you can either pass an HTML `<img>` object (as we do with this type), or actual pixel data (which we do using `image.NRGBA` / `image.RGBA`).